### PR TITLE
Update EM Abstract.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
added reactive power monitoring points.
phase1_reactive_power_accumulator
phase2_reactive_power_accumulator
phase3_reactive_power_accumulator

### CODE REVIEW

**Comment by tasodorff** [comment]
validator errors

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -506,6 +506,9 @@ literals:
 - phase3_powerfactor_sensor
 - powerfactor_sensor
 - reactive_power_sensor
+- phase1_reactive_power_accumulator
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:509]: Please confirm if this is actually an `accumulator` or if it should be a `sensor`

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Checked and updated to sensor.

Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Updated to sensor

**Review by tasodorff — APPROVED** [review]

**Comment by mschulze17** [comment]
@cstirdivant yours to merge :)

**Pull request merged**